Makefile

```diff
@@ -233,6 +238,7 @@ all-fedora: \
 	fixtures/rpm-with-non-utf-8 \
 	fixtures/rpm-alt-layout \
 	fixtures/rpm-with-modules \
+	fixtures/rpm-modular \
 	fixtures/rpm-with-modules-modified \
 	fixtures/rpm-modules-static-context \
```

Comment:
Let's plan to get rid of some of these older fixture repos once the tests have been updated.

Reply:
I concur.